ts/src/gemini.ts

```diff
@@ -297,6 +297,7 @@ export default class gemini extends Exchange {
                         'quote': 'USD',
                     },
                 },
+                'brokenPairs': [ 'efilusd', 'maticrlusd', 'maticusdc', 'eurusdc', 'maticgusd', 'maticusd', 'efilfil', 'eurusd' ],
```

Comment:
@ttodua you sure `maticusdc` isn't valid? same goes for maticusg/etc. they seem matic/STABLE markets

Reply:
@carlosmiei ^